tests/Unit/PointwiseFunctions/AnalyticSolutions/Elasticity/Test_BentBeam.cpp

```diff
@@ -73,27 +74,33 @@ SPECTRE_TEST_CASE(
           79.36507936507935, 38.75968992248062}};
 
   tnsr::I<DataVector, 2> x{{{{1., 2.}, {2., 1.}}}};
-  const auto solution_vars = variables_from_tagged_tuple(
-      solution.variables(x, tmpl::list<Elasticity::Tags::Displacement<2>,
-                                       Elasticity::Tags::Stress<2>>{}));
-  Variables<tmpl::list<Elasticity::Tags::Displacement<2>,
-                       Elasticity::Tags::Stress<2>>>
+  const auto solution_vars = variables_from_tagged_tuple(solution.variables(
+      x,
+      tmpl::list<Elasticity::Tags::Displacement<2>, Elasticity::Tags::Strain<2>,
+                 Elasticity::Tags::Stress<2>>{}));
+  Variables<
+      tmpl::list<Elasticity::Tags::Displacement<2>, Elasticity::Tags::Strain<2>,
+                 Elasticity::Tags::Stress<2>>>
       expected_vars{2};
   auto& expected_displacement =
       get<Elasticity::Tags::Displacement<2>>(expected_vars);
   get<0>(expected_displacement) = DataVector{-0.12, -0.12};
   get<1>(expected_displacement) = DataVector{-0.1227, -0.0588};
+  auto& expected_strain = get<Elasticity::Tags::Strain<2>>(expected_vars);
+  get<0, 0>(expected_strain) = DataVector{-0.12, -0.06};
+  get<1, 0>(expected_strain) = DataVector{0., 0.};
+  get<1, 1>(expected_strain) = DataVector{0.0348, 0.0174};
   auto& expected_stress = get<Elasticity::Tags::Stress<2>>(expected_vars);
   get<0, 0>(expected_stress) = DataVector{12., 6.};
   get<1, 0>(expected_stress) = DataVector{0., 0.};
   get<1, 1>(expected_stress) = DataVector{0., 0.};
   CHECK_VARIABLES_APPROX(solution_vars, expected_vars);
 
-  pypp::check_with_random_values<1,
-                                 tmpl::list<Elasticity::Tags::Displacement<2>,
-                                            Elasticity::Tags::Stress<2>>>(
+  pypp::check_with_random_values<
+      1, tmpl::list<Elasticity::Tags::Displacement<2>,
+                    Elasticity::Tags::Strain<2>, Elasticity::Tags::Stress<2>>>(
       &BentBeamProxy::field_variables, solution, "BentBeam",
-      {"displacement", "stress"}, {{{-5., 5.}}},
+      {"displacement", "strain", "stress"}, {{{-5., 5.}}},
```

Comment:
It would be good to add a test that compares a numerical derivative of the displacement to the implemented strain, i.e. check that the solution numerically solves the elasticity system (now that it's merged). You can use `FirstOrderEllipticSolutionsTestHelpers::verify_solution` for that.

Reply:
Done.